### ISSUE TITLE
Admin\Contacts - External user can Invite user by email

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1715]
#### Description:
As an external user, I want to invite other people via email to access my operator 

#### Acceptance Criteria:

Given I am on the contacts grid #1714 
When I click Invite user
Then I am brought to the invite user page 
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20761-451&t=JxAFAFRUWj85hRmK-4)

Given I am on the invite user page
When I have inputted a valid email address and clicked 'invite'
Then BCIERS will send an automated email to that valid email address inviting them to sign up for BCIERS as a user to access to my operator
I will see a success snackbar that says "the invite email has successfully been sent" 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by patriciarussellCAS** [comment]
I'm wondering if we can de-prioritize this feature for now. @shon-button what are your thoughts? 
Do users have other ways to "invite" additional users? I assume that they just need to do this offline for now - which I'm okay with, but let me know if you feel like this functionality is critical.

**Comment by shon-button** [comment]
> I'm wondering if we can de-prioritize this feature for now. @shon-button what are your thoughts? Do users have other ways to "invite" additional users? I assume that they just need to do this offline for now - which I'm okay with, but let me know if you feel like this functionality is critical.

@patriciarussellCAS 

I agree this feature could be de-prioritize especially since, as you mention, there is a manual invitation work-around.

Note: we do have emailing functionality already implemented in other features; so, this feature may not be too difficult to implement - at the appropriate time.

**Comment by patrickisaac** [comment]
Closing as per backlog day!
